lib/matplotlib/axes/_axes.py

```diff
@@ -1883,96 +1883,83 @@ def bar(self, *args, **kwargs):
-            *width* and ``align='edge'``
+            - 'center': Center the base on the *x* positions.
+            - 'edge': Align the left edges of the bars with the *x* positions.
+
+            To align the bars on the right edge pass a negative *width* and
+            ``align='edge'``.
 
         Returns
         -------
-        bars : matplotlib.container.BarContainer
-            Container with all of the bars + errorbars
+        `matplotlib.container.BarContainer`
+            Container with all the bars and optionally errorbars.
 
         Other Parameters
         ----------------
         color : scalar or array-like, optional
-            the colors of the bar faces
+            The colors of the bar faces.
 
         edgecolor : scalar or array-like, optional
-            the colors of the bar edges
+            The colors of the bar edges.
 
         linewidth : scalar or array-like, optional
-            width of bar edge(s). If None, use default
-            linewidth; If 0, don't draw edges.
-            default: None
+            Width of the bar edge(s). If 0, don't draw edges.
 
         tick_label : string or array-like, optional
-            the tick labels of the bars
-            default: None
+            The tick labels of the bars.
+            Default: None (Use default numeric labels.)
+
+        xerr, yerr : scalar or array-like of shape(N,) or shape(2,N), optional
+            If not *None*, add horizontal / vertical errorbars to the bar tips.
+            The values are +/- sizes relative to the data:
 
-        xerr : scalar or array-like, optional
-            if not None, will be used to generate errorbar(s) on the bar chart
-            default: None
+            - scalar: symmetric +/- values for all bars
+            - shape(N,): symmetric +/- values for each bar
+            - shape(2,N): separate + and - values for each bar
 
-        yerr : scalar or array-like, optional
-            if not None, will be used to generate errorbar(s) on the bar chart
-            default: None
+            Default: None
 
-        ecolor : scalar or array-like, optional
-            specifies the color of errorbar(s)
-            default: None
+        ecolor : scalar or array-like, optional, default: 'black'
+            The line color of the errorbars.
 
         capsize : scalar, optional
-           determines the length in points of the error bar caps
-           default: None, which will take the value from the
+           The length of the error bar caps in points.
+           Default: None, which will take the value from the
            ``errorbar.capsize`` :data:`rcParam<matplotlib.rcParams>`.
 
         error_kw : dict, optional
-            dictionary of kwargs to be passed to errorbar method. *ecolor* and
-            *capsize* may be specified here rather than as independent kwargs.
+            Dictionary of kwargs to be passed to the `~.Axes.errorbar()`
```

Comment:
I think we tend not to include parentheses explicitly (and it would be better to rely on http://www.sphinx-doc.org/en/1.6.5/config.html#confval-add_function_parentheses to set the default doc-wide anyways).  Same comment applies throughout.

Reply:
I don't agree with this. Explicitly indicating that `fill_between` is a function has a value. This should be visible also in plain textual docs, e.g. when viewing it in the terminal.

I propose to discuss the strategy on this and document whatever the result is in https://github.com/matplotlib/matplotlib/blob/master/doc/devel/documenting_mpl.rst Should I open an issue on this?

Reply:
I don't mind discussing the change (well I don't like it but that's another point), but for now grepping for
```
\(\)`
```
and manually exluding the cases where it is followed by a second backquote (i.e., actual code samples) in lib/ yields less than 10 cases, whereas I'm pretty certain there's many more cases without parentheses.  So unless we agree to switch styles, I'd stick to the version without parentheses.